src/main/java/PageObjects/BookCart_RegistrationPage.java

```diff
@@ -63,4 +65,9 @@ public void ClickingRegisterButton() {
  		waitForElementToBeVisibile(RegisterButton);
 		driver.findElement(RegisterButton).click();
 	}
+ 	
+ 	public boolean DisplayingUserNameIsNotAvailable() {
```

Comment:
Use proper naming conventions
Method name should be start by a small letter case